canonical/ubuntu.com
# Wd-12264 Replace account type channel with hasChannelStoreAccess

## Done

- Replace account type channel with hasChannelStoreAccess

## QA

Log in with a channel account 
1. Go to /pro/subscribe, /pro/dashboard
2. Check access is forbidden
3. Go to /pro/distributor
4. Check the page is accessible  

Log in with a non-channel-account
1. Go to /pro/distributor
2. Check k access is forbidden
3. Go to /pro/subscribe, /pro/dashboard
4. Check the pages are accessible

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-12264
